test/run

```diff
@@ -16,4 +16,26 @@ else
   test_dirs=( "$1" )
 fi
 
-exec $bats_executable ${CI:+--tap} "${test_dirs[@]}"
+if command -v parallel &> /dev/null
```

Comment:
Doesn't check that the _GNU_ version of `parallel` is installed, rather use:

```sh
if command -v parallel &> /dev/null \
  && parallel -V &> /dev/null \
  && { parallel -V 2> /dev/null | grep -q '^GNU\>'; }
then
```

Reply:
I'll try to add that now - thanks for the update!

Reply:
Added this code in my latest commit - thanks again!